src/Composer/Package/BasePackage.php

```diff
@@ -22,12 +22,15 @@
  */
 abstract class BasePackage implements PackageInterface
 {
+    /**
+     * @phpstan-var array<string, array<description: string, method: Link::TYPE_*>>
```

Comment:
hmm I guess I had a typo in here

```suggestion
     * @phpstan-var array<string, array{description: string, method: Link::TYPE_*}>
```

Reply:
yeah, your are right https://phpstan.org/writing-php-code/phpdoc-types#array-shapes

will fix this

Reply:
I'm wondering why PHPStan does not complain about the syntax? :thinking:

Reply:
> I'm wondering why PHPStan does not complain about the syntax?

it seems phpstan only errors when the level is 2 or higher, see
https://phpstan.org/r/b8878f4d-af86-4d0c-a39b-8fabb4811980

Reply:
oh okay, didn't recognized the low level in the PHPStan config 🙈